KAS-1756: Login time

sergiofenoll opened the pull request:
# :warning: This PR merges into #324 
Either merge this one before merging the parent PR, or change the base branch.

sergiofenoll committed:
feat: add login-activity resource

sergiofenoll committed:
feat: add created property on users

sergiofenoll committed:
chore: remove unneeded type constraint in mu-auth

sergiofenoll committed:
Merge branch 'feature/KAS-3608-blocking' into feature/KAS-1756-login-time

erikap commented on config/authorization/config.ex:
```diff
@@ -296,6 +301,20 @@ defmodule Acl.UserGroups.Config do
+        access: access_by_role( admin_roles() ),
+        graphs: [
+          %GraphSpec{
+            graph: "http://mu.semte.ch/graphs/system/users",
```
Als we toch voor de graph `http://mu.semte.ch/graphs/system/users` gaan (wat OK is voor mij aangezien we specifieke subclass `ext:LoginActivity` definieren), zou ik geen aparte group-spec maken en dit toevoegen aan de bestaande `admin` group-spec. Mu-auth laat dan ook schrijfrechten toe, wat mij niet problematisch lijkt voor admin gebruikers, maar dispatcher laat vanuit FE sowieso enkel `GET` requests door.

Note voor toekomst: als group-specs aangepast worden, vereist dit ook altijd aanpassing van de geconfigureerde eager indexing groups in mu-search config en cache-warmup.

sergiofenoll committed:
feat: add position to user roles

sergiofenoll committed:
chore: remove unnecessary GroupSpec

erikap committed:
Add clarification on relation between user and login-activity

ValenberghsSven committed:
Merge pull request #326 from kanselarij-vlaanderen/feature/KAS-3607-filtering-users

KAS-3607: Filtering users

sergiofenoll committed:
chore: remove user-management-service

sergiofenoll committed:
chore: set delta consuming services to sink instead of disabling them

This should help with the lack of deltas, which is because the delta
notifier has a long wait time when it can't resolve on of its
destinations. If the services are enabled (but do nothing, due to being
simple sinks), the delta notifier can work in a timely manner.

erikap committed:
Merge pull request #327 from kanselarij-vlaanderen/feature/KAS-3609-users-code-cleanup

KAS-3609: Users code cleanup

erikap committed:
Bump login and mock-login service

erikap committed:
Merge branch 'development' into feature/KAS-1756-login-time
